docs/book/user-guide/advanced-guide/advanced-guide.md

```diff
@@ -2,10 +2,10 @@
+<table data-card-size="large" data-view="cards"><thead><tr><th></th><th></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td><mark style="color:purple;"><strong>Pipelining features</strong></mark></td><td>Detailed look at how to configure and handle ZenML steps and pipelines.</td><td><a href="pipelining-features/pipelining-features.md">pipelining-features.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Artifact management</strong></mark></td><td>Managing your input and output data with ZenML.</td><td><a href="data-management/data-management.md">artifact-management.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Model management</strong></mark></td><td>Managing your models with ZenML.</td><td><a href="data-management/model-management.md">model-management.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Environment management</strong></mark></td><td>Navigating multiple development environments.</td><td><a href="environment-management/environment-management.md">environment-management.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Secret management</strong></mark></td><td>Registering and utilizing secrets with ZenML.</td><td><a href="secret-management/secret-management.md">secret-management.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Custom stack solutions</strong></mark></td><td>Developing custom stack solutions through ZenML base abstractions.</td><td><a href="custom-stack-solutions/custom-stack-solutions.md">custom-stack-solutions.md</a></td></tr></tbody></table>
```

Comment:
The cards here do not reflect the actual sections in the guide. Not sure if this is intended or not.